cpu/riscv_common/Makefile.features

```diff
@@ -1,3 +1,5 @@
+CPU_ARCH := risc-v
```

Comment:
Same for this one, can you add to a commit where you say "adding missing arch or something like that", or fix missing arch and you can ad comment on the `:=` change in that commit as well.